pom.xml

```diff
@@ -68,7 +68,7 @@ under the License.
 
   <properties>
     <javaVersion>8</javaVersion>
-    <mavenVersion>3.2.5</mavenVersion>
+    <mavenVersion>3.9.6</mavenVersion>
```

Comment:
I don't remember noticing this. If I had I would have inquired further. Is 3.9.6 required to compile? If so, why?